src/platform/site-wide/user-nav/components/SearchHelpSignIn.jsx

```diff
@@ -51,6 +54,7 @@ class SearchHelpSignIn extends React.Component {
         )}
         {isSubdomain && (
           <a
+            onClick="window.dataLayer.push({ 'event': 'nav-jumplink-click' });"
```

Comment:
Can we import and use that `recordEvent` function here too? 

`recordEvent` is also globally available in templates as well, so it should always work as an alias.

Reply:
Np, @ncksllvn - updated.